enterprise/models.py

```diff
@@ -383,6 +383,13 @@ def identity_providers(self):
         """
         return self.enterprise_customer_identity_providers.all()
 
+    @property
+    def identity_provider_ids(self):
+        """
+        Return the identity provider Ids associated with this enterprise customer.
+        """
+        return str(self.identity_providers.values_list('provider_id', flat=True))
```

Comment:
Shouldn't we return a list rather than a string?

Reply:
@irfanuddinahmad good catch